packages/split-button/src/SplitButton.ts

```diff
@@ -139,6 +140,7 @@ export class SplitButton extends SizedMixin(PickerBase) {
             `,
             html`
                 <sp-button
+                    aria-describedby=${this.description}
```

Comment:
This ones hard to make decisions about, as it's deprecated. Hard to say whether we should be describing both buttons in this way or not.

Reply:
@Rajdeepc can you help me out with this?